app/controllers/users/registrations_controller.rb

```diff
@@ -36,17 +36,14 @@ def index
   def show
     @user = User.find(params[:id])
 
-    if @user.blank?
```

Comment:
Why this change?

Reply:
I think the line `@user = User.find(params[:id])` [either returns a User from our db or returns an error if a User for the given id does not exist](https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find). Therefore there won't be a situation where `@user` is ever blank/nil currently so this code is redundant.

Reply:
Oh right. Can you change that and use User.where(id: params[:id]).last to avoid the rescue error.

Reply:
Sure I'll make the change now 👍 Is there a reason we don't want to use a rescue block?

Reply:
Not a big fan of rescue blocks for this situation.

Reply:
Updated